README.md

```diff
@@ -108,7 +144,6 @@ CSS
 }
 ```
 
-**Fixes for Filter and Filter+ modes**.
```

Comment:
Had been moved to line 122 so it will be understandable and more clearly sections.